pypeit/wavetilts.py

```diff
@@ -388,6 +572,30 @@ def run(self, maskslits=None, doqa=True, debug=False, show=False):
             thismask_science = self.slitmask_science == slit
             self.final_tilts[thismask_science] = self.tilts[thismask_science]
 
+        if debug:
```

Comment:
At some point I also had some code in ginga that plotted the tilts on the image. This is probably how we want to do this.

Reply:
Here's what this looks like for the VLT XShooter NIR data.

![nir_line_trace](https://user-images.githubusercontent.com/15785898/68430725-72f1df00-0165-11ea-8d30-1c829cd61b26.png)

Reply:
This looks good except the second half of the K-band has no lines detected. I looked back at some old QA and this seems to also have been the case before. I think we need to change the line detection threshold or whatever we are doing to keep the lines. It may be that this stuff is way too conservative.

Reply:
I looked at the line detection plots for the last order, and we are being a bit conservative, but it's just harder to identify good lines in the reddest order.  I think an iteration can help, but we may also need a slit-/order-dependent line detection threshold.

Reply:
Yes, I like an order dependent line detection threshold. That sounds like the way to go.